tests/roles/manila_adoption/tasks/main.yaml

```diff
@@ -36,6 +36,47 @@
         {{ oc_header }}
         oc patch openstackcontrolplane openstack --type=merge --patch-file=/tmp/manila_cephfs.yaml
 
+- name: Deploy Podified Manila - Netapp
+  when: manila_backend == "netapp"
+  block:
+    # Note: I might need to replace this task with something more complex where
+    # 1. ssh into controller node
+    # 2. extract the netapp config from manila.conf
+    # 3. populate the ansible variables required to build netapp-secret.conf
+    - name: Fail if netapp input are not defined
+      when: |
+        netapp_host is not defined or
```

Comment:
@jistr adding you here because I have a question. 
The idea here is to provide `netapp_*` parameters as input to the playbook and then perform the regular adoption without doing so much complex operations to retrieve that data.
Am I breaking a pattern? 
I see services like `Barbican` do something like [1], but I thought it was better at "job" level to pass this information as input or retrieve it at a different layer.
Is there a pattern that has been established and we want to follow? How the existing unijob deals with secret creation and these kind of information?

[1] https://github.com/openstack-k8s-operators/data-plane-adoption/blob/ef823f2f2c5772fce9e7f04876e0bdfe049deb3a/tests/roles/barbican_adoption/tasks/main.yaml#L6